UE/Application/Ports/UserPort.cpp

```diff
@@ -37,6 +43,48 @@ void UserPort::showConnected()
     menu.clearSelectionList();
     menu.addSelectionListItem("Compose SMS", "");
     menu.addSelectionListItem("View SMS", "");
+    gui.setAcceptCallback([&](){
+        if(menu.getCurrentItemIndex().second == 1){
+            showSmsList();
+        };
+    });
+}
+
+void UserPort::showSmsList() {
+
+    IUeGui::IListViewMode& menu = gui.setListViewMode();
+    menu.clearSelectionList();
+    std::vector<Sms> ListSms = db.getAll();
+    if(ListSms.empty()){
+        menu.addSelectionListItem("No SMS in DB", "");
+
+    } else {
+        for(auto OneSms : ListSms){
+            menu.addSelectionListItem("From: "+to_string(OneSms.from),OneSms.text);
+        }
+
+    }
```

Comment:
Add white space around braces and `+`, start variables with lower case.
```suggestion
    if(ListSms.empty()) {
        menu.addSelectionListItem("No SMS in DB", "");
    } else {
        for(auto sms : ListSms) {
            menu.addSelectionListItem("From: " + to_string(sms.from), sms.text);
        }
    }
```